src/bar/BarChartWithFilters.vue

```diff
@@ -10,18 +10,18 @@
                                  v-model="indicatorId"
                                  :normalizer="normalizeIndicators"></tree-select>
                 </div>
-                <div v-if="!(xAxisConfig && xAxisConfig.fixed)" id="x-axis-fg" class="form-group">
+                <div v-if="!xAsisIsFixed" id="x-axis-fg" class="form-group">
```

Comment:
Sorry, hadn't noticed this was a typo too
```suggestion
                <div v-if="!xAxisIsFixed" id="x-axis-fg" class="form-group">
```